it_zwierzak/adoption_announcement/views.py

```diff
@@ -1,22 +1,45 @@
 from django.shortcuts import render
-from django.http import HttpResponseRedirect, HttpResponse
+from django.views.generic import FormView
+from .forms import AdoptionForm
+from django.contrib.auth.decorators import login_required
+from django.contrib.auth.mixins import LoginRequiredMixin
 
-from .templates import forms
 
-def index(request):
-    return HttpResponse("Ogłoszenie Adopcji")
-
-def base(request):
-    return render(request, 'adoption_announcement/base.html')
 
+# zaadoptowane
```

Comment:
Po pierwsze to po angielsku komentarze do kodu, po dwa to skoro nazwa widoku to `adopted`, a zaadaptowane to tylko tłumaczenie z ang na pl to raczej ten komentarz jest zbyteczny, jeśli jednak bardzo chcemy dodać, to w postaci docstringa, czyli tak:
```python
def some_view(request):
    """I'm a docstring, explain my purpose here!"""
```

Reply:
usunięte